air-quality-backend/system_tests/api_suite/measurements_summary_api_validation_tests.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+
+import pytest
+import requests
+
+from system_tests.utils.api_utilities import format_datetime_as_string
+from system_tests.utils.routes import Routes
+
+base_url = Routes.measurement_summary_api_url
+location_type = "city"
+measurement_base_time_string_24_7_20_14_0_0 = format_datetime_as_string(
+    datetime.datetime(2024, 7, 20, 14, 0, 0, tzinfo=datetime.timezone.utc),
+    "%Y-%m-%dT%H:%M:%S+00:00",
+)
+measurement_time_range = 90
+
+success_status_code = 200
+validation_error_status_code = 422
+
+
+@pytest.mark.parametrize(
+    "payload, expected_status_code",
+    [
+        (
+            {
+                "location_type": location_type,
+                "measurement_base_time": measurement_base_time_string_24_7_20_14_0_0,
+                "measurement_time_range": measurement_time_range,
+            },
+            success_status_code,
+        ),
+        (
+            {
+                "location_type": "",
+                "measurement_base_time": measurement_base_time_string_24_7_20_14_0_0,
+                "measurement_time_range": measurement_time_range,
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "location_type": location_type,
+                "measurement_base_time": "",
+                "measurement_time_range": measurement_time_range,
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "location_type": location_type,
+                "measurement_base_time": measurement_base_time_string_24_7_20_14_0_0,
+                "measurement_time_range": "",
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "location_type": "",
+                "measurement_base_time": "",
+                "measurement_time_range": "",
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "measurement_base_time": measurement_base_time_string_24_7_20_14_0_0,
+                "measurement_time_range": measurement_time_range,
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "location_type": location_type,
+                "measurement_time_range": measurement_time_range,
+            },
+            validation_error_status_code,
+        ),
+        (
+            {
+                "location_type": location_type,
+                "measurement_base_time": measurement_base_time_string_24_7_20_14_0_0,
+            },
+            validation_error_status_code,
+        ),
+        (
+            {},
+            validation_error_status_code,
+        ),
+    ],
+)
+def test__required_parameter_combinations__verify_response_status_code(
```

Comment:
I not sure you gain much from including the success case in the parametrization. We might be better served by one test that calls it successfully without parameters, followed by a parameterised test for all the failures. You wouldn't need to include status code as a param, and it would I think aid readability of what each test is up to.
The name could also be improved after this as well.